presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -10,16 +10,102 @@ it does not cause running queries to fail; instead new queries become queued.
 A resource group may have sub-groups or may accept queries, but may not do both.
 
 The resource groups and associated selection rules are configured by a manager, which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+
+There are two built-in managers: file-based and database-based. Add an ``etc/resource-groups.properties``
```

Comment:
You can use a file-based or a database-based resource group manager. Add a file ``etc/resource-groups.properties`` to enable and configure the desired manager.